[Bugfix 18392] Improvements to "ceiling" function documentation

peter-b opened the pull request:
Renamed documentation name to be the full function name `ceiling` instead of the abbreviated synonym `ceil`. Some other formatting tweaks.

This is a rebased version of #4914

bhall2001 committed:
[Bug 18392] Improvements to "ceiling" function documentation.

- Renamed documentation name to be the full function name `ceiling`
  instead of the abbreviated synonym `ceil`.

peter-b committed:
docs: Formatting tweaks to ceiling & floor function docs

- Mark example expressions as inline code
- Remove an extra period

livecodeali commented on docs/dictionary/function/ceiling.lcdoc:
```diff
@@ -34,11 +34,10 @@ Description:
-<return(glossary)|returns> 6, but ceiling(4.99) <return(glossary)|returns>
-
-5. 
+The <ceiling> function does not always <return(constant)> the
```
This should be `<return(glossary)>` or even just `<return>` - not sure how `return (constant)` managed to sneak into the references here

peter-b committed:
Remove references to return constant from floor docs

In all cases, the `floor()` docs were referring to the `return` entry
in the glossary, not to the `return` constant.

peter-b commented:
@livecodeali Thank you for spotting that; I've fixed it.

livecodeali commented:
@peter-b The same problem exists in the `floor` doc (sorry, I only just realised I wasn't looking at the whole changeset!)

Also you may want to tweak the examples to use `put` as they don't currently compile (but I suspect that may be a wider problem with function docs)

livecodeali commented:
@livecode-vulcan review ok 486868d

livecode-vulcan commented:
💙 review by @livecodeali ok 486868d3ac6f879b522bef505a64b8793589018f

livecode-vulcan commented:
😎 test success 486868d3ac6f879b522bef505a64b8793589018f

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/634)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/639)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/634)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/638)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/643)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/635)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/635)
